mlflow/mlflow
# SageMaker Deployment: Allow users to specify VPC configurations for model creation

This addresses #286.

**Test Plan**

The following test cases will be executed manually:

1. Deploy a model to SageMaker with `vpc_config=None`.
**Expected Result**: The model is deployed successfully in the same fashion as it was prior to the introduction of the `vpc_config` parameter.

2. Deploy a model to SageMaker while specifying a valid VPC configuration
**Expected Result**: The new SageMaker model should be created inside the specified VPC; this should be indicated in the "Models" section of the SageMaker UI.
